android/src/main/java/com/thebluealliance/androidclient/datafeed/Database.java

```diff
@@ -435,11 +416,11 @@ public long storeResponse(String url, String updated) {
             cv.put(Response.URL, url);
             cv.put(Response.LASTUPDATE, updated);
             cv.put(Response.LASTHIT, new Date().getTime());
-            return safeInsert(TABLE_API, null, cv);
+            return mDb.insert(TABLE_API, null, cv);
         }
 
         public APIResponse<String> getResponse(String url) {
-            Cursor cursor = safeQuery(TABLE_API, new String[]{Response.URL, Response.LASTUPDATE, Response.LASTHIT},
+            Cursor cursor = mDb.query(TABLE_API, new String[]{Response.URL, Response.LASTUPDATE, Response.LASTHIT},
```

Comment:
Perhaps refactor to ```getResponseIfExists()``` and remove ```responseExists()```. The only time I see us using ```responseExists()``` is to check if a response exists before trying to retrieve it; combining all of that into one step would save precious CPU cycles.